src/styles/styles.js

```diff
@@ -993,13 +993,15 @@ const styles = {
         width: variables.componentSizeNormal,
         backgroundColor: themeColors.icon,
         borderRadius: variables.componentSizeNormal,
+        pointerEvents: 'none',
     },
 
     avatarSmall: {
         height: variables.avatarSizeSmall,
         width: variables.avatarSizeSmall,
         backgroundColor: themeColors.icon,
         borderRadius: variables.avatarSizeSmall,
+        pointerEvents: 'none',
```

Comment:
heads up, this might be working on web but it is throwing an invalid style warning on Android and should be fixed. You can see which style props are supported by `Image` here -> https://reactnative.dev/docs/image-style-props#props

<img width="339" alt="2021-05-25_08-07-40" src="https://user-images.githubusercontent.com/32969087/119547015-47771b00-bd30-11eb-86d6-dd52f0b2d7ea.png">

`View` has the `pointerEvents` prop that we can use to achieve a similar effect

https://reactnative.dev/docs/view#pointerevents